1.8/development/index.md

```diff
@@ -1,16 +1,103 @@
 
-All services in the Universe are required to meet a standard as defined by the DC/OS project team. For details on publishing a release of a DC/OS service, see [Publish a Package][2] in the Universe documentation.
+# <a name="universe"></a>Package Repositories
 
-For detailed information about the JSON files of a package, see the [Universe][1] docs.
+The DC/OS Universe contains all of the services that are installable DC/OS. For more information on DC/OS Universe, see the [GitHub Universe repository(https://github.com/mesosphere/universe).
 
- [1]: http://mesosphere.github.io/universe/
- [2]: http://mesosphere.github.io/universe/#publish-a-package-1
+All services in the package repositories are required to meet a certain standard as defined by Mesosphere. For details on submitting a DC/OS service, see [Contributing a package](http://mesosphere.github.io/universe/#publish-a-package-1).
+
+# <a name="adminrouter"></a>Admin Router and web interface integration
+
+By default, a DC/OS service is deployed on a [private agent node](/1.8/overview/concepts/#private-agent-node). To allow configuration control or monitoring of a service by a user, the admin router proxies calls on the master node to the service in a private node on the cluster. The HTTP service endpoint requires relative paths for artifacts and resources. The service endpoint can provide a web interface, a RESTful endpoint, or both. When creating a DC/OS CLI subcommand it is common to have a RESTful endpoint to communicate with the scheduler service.
+
+The integration to the admin router is automatic when a framework scheduler registers a `webui_url` during the registration process with the Mesos master. There are a couple of limitations:
+
+*   The URL must NOT end with a backslash (/). For example, this is good `internal.dcos.host.name:10000`, and this is bad `internal.dcos.host.name:10000/`.
+*   DC/OS supports 1 URL and port.
+
+When the `webui_url` is provided, the service is listed on the DC/OS web interface as a service with a link. That link is the admin router proxy URL name that is based on a naming convention of: `/service/<service_name>`. For example, `<dcos_host>/service/unicorn` is the proxy to the `webui_url`. If you provide a web interface, it will be integrated with the DC/OS web interface and users can click the link for quick access to your service.
+
+Service health check information is provided from the DC/OS service tab when:
+
+*   There are service health checks defined in the `marathon.json` file. For example:
+
+>      "healthChecks": [
+>      {
+>        "path": "/",
+>        "portIndex": 1,
+>        "protocol": "HTTP",
+>        "gracePeriodSeconds": 5,
+>        "intervalSeconds": 60,
+>        "timeoutSeconds": 10,
+>        "maxConsecutiveFailures": 3
+>      
+>     
```

Comment:
This is rendered as code *and* blockquoted. Is the blockquote necessary?

Might want to adjust indentation and add closing delimiters, too:

```
        "healthChecks": [
          {
            "path": "/",
            "portIndex": 1,
            "protocol": "HTTP",
            "gracePeriodSeconds": 5,
            "intervalSeconds": 60,
            "timeoutSeconds": 10,
            "maxConsecutiveFailures": 3
          }
        ]
```